index.html

```diff
@@ -49,10 +49,21 @@
 		<div class="col-sm-3"></div>
 	
 		<div class="col-sm-6" id="picColumn">
-		<button type="button" class="btn btn-primary-outline" id="crystal1_span"><img src="./assets/images/crystal1.png" class="img-rounded" alt="Crystal 1" width="100" height="100"></button>
-		<button type="button" class="btn btn-primary-outline" id="crystal2_span"><img src="./assets/images/crystal2.png" class="img-rounded" alt="Crystal 2" width="100" height="100"></button>
-		<button type="button" class="btn btn-primary-outline" id="crystal3_span"><img src="./assets/images/crystal3.png" class="img-rounded" alt="Crystal 3" width="100" height="100"></button>
-		<button type="button" class="btn btn-primary-outline" id="crystal4_span"><img src="./assets/images/crystal4.png" class="img-rounded" alt="Crystal 4" width="100" height="100"></button>
+			<button type="button" class="btn btn-primary-outline crystal-btn" id="crystal1_span">
```

Comment:
I just added a generic `crytal-btn` class to the 4 buttons so as to set the 1 event listener on all of them. Also, when you start having long lines of html (or code in general) feel free to break it into multiple lines as it will help you (and other developers) parse through the code much more quickly.